### ISSUE TITLE
A Copilot response containing one or more code blocks, each code block is followed by a nameless toolbar: A11y_Visual Studio Code Client_Copilot chat_Devtools

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#237794]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

## GitHubTags:
#A11yeDAD; #A11yTCS; #A11ySev3; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG4.1.2; #Devtools;

## Environment and OS details:
Visual Studio Code insider version 1.96.2
Microsoft Windows 11 Enterprise (24H2)
OS Build 26100.2605

## Repro Steps:
1.Open Visual studio code insider
2.Link a GitHub Account
3. Sign Up for Free Copilot Use
4. Sign Up for Free Copilot Use and in copilot chat copilot response containing one or more code blocks. 
5. TAB till tool bar of each code block and inspect it.
6. Observe that a copilot response containing one or more code blocks, each code block is followed by a nameless toolbar.

## Actual:
A copilot response containing one or more code blocks, each code block is followed by a nameless toolbar.

## Expected:
Name should provide for each toolbar container that includes the name of the associated code block.

## User Impact:
Screen reader user will face difficulty, they cannot differentiate between the toolbars.

## Attachment:

![Image](https://github.com/user-attachments/assets/8bba2d5f-e517-4ac9-8d5f-ffd15f1dac0e)

**Comment by kapilvaishna** [comment]
issue is not repro in below environment so closing the bug.

Version: 1.98.2 (user setup)
Commit: ddc367ed5c8936efe395cffeec279b04ffd7db78
Date: 2025-03-12T13:32:45.399Z
Electron: 34.2.0
ElectronBuildId: 11161602
Chromium: 132.0.6834.196
Node.js: 20.18.2
V8: 13.2.152.36-electron.0
OS: Windows_NT x64 10.0.26100 

![Image](https://github.com/user-attachments/assets/5ac2a7f4-1dfe-47f0-9eef-5ebffaed995e)### 


![Image](https://github.com/user-attachments/assets/b38f3c31-6db5-4293-b9a9-ef9e3a81b271)
